content/content-tool/tool/src/webapp/vm/content/sakai_resources_list.vm

```diff
@@ -17,10 +20,13 @@
   }
   .truncate{
     width: 180px;
-    white-space: nowrap;
+    white-space: nowrap; 
     overflow: hidden;
     text-overflow: ellipsis;
   }
+  .text-truncate{
+    white-space: normal !important;
```

Comment:
hi @Sharadhi98 is it possible to avoid using !important and rely on specificity?

Reply:
note that text-truncate is a bootstrap class. i don't think it makes sense to override Bootstrap like this. if you don't like the use of text-truncate, the use of the class should be removed/altered.

Reply:
Thank you for your comment. I will go through the code and try to avoid !important.